command/scaling_policy_list.go

```diff
@@ -126,23 +122,30 @@ func (s *ScalingPolicyListCommand) Run(args []string) int {
 		return 0
 	}
 
+	output := formatScalingPolicies(policies)
+	s.Ui.Output(output)
+	return 0
+}
+
+func formatScalingPolicies(stubs []*api.ScalingPolicyListStub) string {
+	if len(stubs) == 0 {
+		return "No policies found"
+	}
+
 	// Create the output table header.
-	output := []string{"ID|Enabled|Type|Target"}
+	policies := []string{"ID|Enabled|Type|Target"}
 
 	// Sort the list of policies based on their target.
-	sortedPolicies := scalingPolicyStubList{policies: policies}
+	sortedPolicies := scalingPolicyStubList{policies: stubs}
 	sort.Sort(sortedPolicies)
 
 	// Iterate the policies and add to the output.
 	for _, policy := range sortedPolicies.policies {
-		output = append(output, fmt.Sprintf(
+		policies = append(policies, fmt.Sprintf(
 			"%s|%v|%s|%s",
 			policy.ID, policy.Enabled, policy.Type, formatScalingPolicyTarget(policy.Target)))
```

Comment:
Not sure if you would want to fix here or later, but we could reduce the ID to the first block since it's an UUID:

```shellsession
$ ./bin/nomad scaling policy list
ID                                    Enabled  Type        Target
94fe7ab5-63b1-d25d-285e-3ff59457996d  true     horizontal  Namespace:default,Job:webapp2,Group:demo

$ ./bin/nomad deployment list
ID        Job ID      Job Version  Status      Description
5adba4fb  webapp2     0            successful  Deployment completed successfully
f693f93d  webapp2     0            successful  Deployment completed successfully
2e85e08d  webapp      15           successful  Deployment completed successfully
9cdb46b3  webapp      14           successful  Deployment completed successfully
212be987  webapp      13           successful  Deployment completed successfully
afcba73a  webapp      12           successful  Deployment completed successfully
6f19ecd3  webapp      11           successful  Deployment completed successfully
c00bd7de  webapp      10           successful  Deployment completed successfully
b0299e46  webapp      9            successful  Deployment completed successfully
af0e331f  webapp      8            successful  Deployment completed successfully
050fe10a  webapp      7            successful  Deployment completed successfully
320793f0  webapp      6            successful  Deployment completed successfully
be444b5c  webapp      5            successful  Deployment completed successfully
3e874047  haproxy     0            successful  Deployment completed successfully
9e195d24  prometheus  0            successful  Deployment completed successfully
```